tool/tsh/tsh.go

```diff
@@ -4579,22 +4592,12 @@ func setEnvFlags(cf *CLIConf, getEnv envGetter) {
 			cf.SiteName = clusterName
 		} else if clusterName = getEnv(siteEnvVar); clusterName != "" {
 			cf.SiteName = clusterName
-		} else if clusterName = getEnv(teleport.SSHTeleportClusterName); clusterName != "" {
```

Comment:
The purpose of `setEnvFlags` is to populate the `cf` from env. Moving this code to a different function breaks that, making it harder to reason about the code. We also lose `getEnv`, which helps with tests.